contracts/token/ERC20/extensions/ConfidentialERC20Mintable.sol

```diff
@@ -29,14 +29,15 @@ abstract contract ConfidentialERC20Mintable is Ownable2Step, ConfidentialERC20 {
     ) Ownable(owner_) ConfidentialERC20(name_, symbol_) {}
 
     /**
-     * @notice Mint tokens.
-     * @param amount Amount of tokens to mint.
+     * @notice          Mint tokens.
+     * @param to        Address to mint tokens to.
+     * @param amount    Amount of tokens to mint.
      */
-    function mint(uint64 amount) public virtual onlyOwner {
-        _unsafeMint(msg.sender, amount);
+    function mint(address to, uint64 amount) public virtual onlyOwner {
+        _unsafeMint(to, amount);
         /// @dev Since _totalSupply is not encrypted and _totalSupply >= balances[msg.sender],
```

Comment:
Please fix this comment to something like "@dev Since _totalSupply is not encrypted and _totalSupply is always increasing in our implementation without burn"